_clients/data-prepper/index.md

```diff
@@ -12,4 +12,77 @@ Data Prepper is a server side data collector capable of filtering, enriching, tr
 
 Data Prepper lets users build custom pipelines to improve the operational view of applications. Two common uses for Data Prepper are trace and log analytics. [Trace analytics]({{site.url}}{{site.baseurl}}/observability-plugin/trace/index/) can help you visualize the flow of events and identify performance problems, and [log analytics]({{site.url}}{{site.baseurl}}/observability-plugin/log-analytics/) can improve searching, analyzing and provide insights into your application.
 
+## Concepts
+
+Data Prepper is compromised of **Pipelines** that collect and filter data based on the components set within the pipeline. Each component is pluggable, enabling you to use your own custom implementation of each component. These components include: 
+
+- One [source](#source)
+- One or more[sinks](#sink)
+- (Optional) One [buffer](#buffer)
+- (Optional) One or more[processors](#processor)
+
+A single instance of Data Prepper can have one or more pipelines. 
+
+Each pipeline definition contains two required components **source** and **sink**. If buffers and processors are missing from the Data Prepper pipeline, Data Prepper uses the default buffer and a no-op processor. 
+
+### Source 
+
+Source is the input component of a pipeline that defines the mechanism through which a Data Prepper pipeline will consume events. A pipeline can have only one source. The source can consume events either by receiving the events over HTTP or HTTPS or reading from external endpoints like OTeL Collector for traces and metrics and S3. Source have their own configuration options based on the format of the events (such as string, json,  cloudwatch logs, or open telemetry trace). The source component consumes events and writes them to the buffer component. 
```

Comment:
Should the first sentence start with "The source component"? In the third sentence, should it read something like "...or by reading from external endpoints, like OTeL Collector for traces, metrics, and Amazon Simple Storage Service (Amazon S3)"?